src/qs8-gemm/MRx4c8-sse.c.in

```diff
@@ -31,17 +31,35 @@ $if DATATYPE != "QD8":
   #include <xnnpack/unaligned.h>
 
 
-$DATATYPE_SPEC = {"QC8": "qs8_qc8w", "QD8": "qd8_f32_qc8w", "QC4": "qd8_f32_qc4w", "QS8": "qs8", "QU8": "qu8"}[DATATYPE]
+$#
```

Comment:
Suggest moving template into src/qd8-f32-qb4w-gemm/ folder

Reply:
Do you mean also suggest that we create a different template for `QB4W`?

Not sure which way it will fall from maintenance point of view TBH, i.e. overloaded template vs. templates with similar logic.

Reply:
wanted to follow up on if this meant moving the SET_INDENT function to some common tools file, or if all the qb4w gemm metakernels should live in their own folder seperate from qs8-gemm